test/image/mocks/axes_breaks-matches.json

```diff
@@ -0,0 +1,71 @@
+{
+    "data": [
+        {
+            "mode": "lines+markers",
+            "x": [
+                "2020-05-09",
```

Comment:
If you look at the baseline for this mock, you can see a bug in the autorange where the left edge gets no padding. This has nothing to do with matching axes, it'll happen on a single axis with rangebreaks, just because the first point is inside a break.